module-apps/apps-common/WindowsPopupFilter.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "WindowsPopupFilter.hpp"
+#include "WindowsStack.hpp"
+#include "data/PopupRequestParams.hpp"
+#include "AppWindow.hpp"
+
+namespace gui::popup
+{
+    void Filter::attachWindowsStack(app::WindowsStack *stack)
```

Comment:
is there an option that we will replace stack here ? with some other stack ? maybe make it const and passed as required in constructor?